src/main/redux/sagas/index.ts

```diff
@@ -61,6 +61,9 @@ export function* rootSaga() {
     // send initSucess first
     yield put(appActions.initSuccess.build());
 
+    // watch all electon exit event
+    yield app.exit();
```

Comment:
This is extremely confusing for people reading / skimming the code (because of Electron's own `app.exit()`), much like when code uses "reserved" NodeJS packages for generic variable names like `path`, `os`, `fs`, etc.

So, would you mind swapping the `import { app as appElectron } from "electron"` around, for example `import * as appSaga from "./app";`? Thank you

Reply:
fixed